app.rb

```diff
@@ -0,0 +1,24 @@
+require "sinatra/base"
+require "./lib/peep"
+
+class Chitter < Sinatra::Base
```

Comment:
This file is very nice, you haven't crammed a bunch of logic into here which is good.